_test/check-exercise-crate.sh

```diff
@@ -22,12 +23,15 @@ fi
 
 TRACK_ROOT="$(git rev-parse --show-toplevel)"
 
-if !(cd "$TRACK_ROOT/$EXERCISE_CRATE_PATH" && cargo check); then
-	echo "\nAn error has occurred while building the exercise crate.\nPlease make it compile."
+if ! (cd "$TRACK_ROOT/$EXERCISE_CRATE_PATH" && cargo check); then
+    echo
+	echo "An error has occurred while building the exercise crate."
+    echo "Please make it compile."
```

Comment:
Odd indentation here and elsewhere. Maybe run a batch tabs to spaces script.

Reply:
Batch applied, then needed to restore the executable bit on each file.